client/src/App.tsx

```diff
@@ -1,33 +1,27 @@
 import * as React from 'react';
 import 'semantic-ui-css/semantic.min.css';
-import { Container, Icon, Modal, Popup } from 'semantic-ui-react';
-import About from './About';
+import { Container } from 'semantic-ui-react';
 import ChannelStatus from './channel/components/ChannelStatus';
 import DatasetList from './dataset/components/DatasetList';
 import ErrorList from './errors/components/ErrorList';
 import logo from './images/LiberTEM logo-medium.png';
+import HeaderMenu from './Menu'
 
 class App extends React.Component {
     public render() {
         return (
+            <div>
```

Comment:
Instead of the `<div></div>`, if you don't need the `<div>` element in the DOM, you can also use `<></>`. Also the contents then need to be re-indented (can possibly be done by re-formatting in vs code?)

Reply:
Thanks for the suggestion, Should have installed a formatter earlier.

Reply:
Just curious, which formatter are you using? That's something different from the built-in TypeScript formatting of vs code, right?

Also, it would be helpful to only format the newly added/changed code (there should be a setting "format on save" or similar, combined with manually applying "Format Selection"), otherwise, when reviewing, I always have to check if a given change is only a formatting change, or part of the "real" changes.

(sadly, automatically formatting only changed lines is not yet supported in vs code, https://github.com/microsoft/vscode/issues/44075)

Reply:
> Just curious, which formatter are you using? That's something different from the built-in TypeScript formatting of vs code, right?

Prettier.Sorry for the trouble, no way to format just the changed lines.(https://github.com/prettier/prettier-vscode/issues/630).

I had no idea about "Format Selection", I will only use that from now on. Thanks.